src/main/java/danta/aem/contextprocessors/AddSlingModelsPropertiesContextProcessor.java

```diff
@@ -56,14 +56,16 @@
 public class AddSlingModelsPropertiesContextProcessor
         extends AbstractCheckComponentCategoryContextProcessor<TemplateContentModel> {
 
+    private static final Set<String> ANY_OF = Collections.unmodifiableSet(Sets.newHashSet(SLING_MODELS_CATEGORY));
+
+    @Reference
+    private ModelFactory modelFactory;
+
     @Override
     public Set<String> anyOf() {
-        return Sets.newHashSet(SLING_MODELS_CATEGORY);
+        return ANY_OF;
     }
 
-    @Reference
-    private ModelFactory modelFactory = null;
-
```

Comment:
Moved field before first method and removed `null` initialization for consistency.